src/settings_data.h

```diff
@@ -1049,7 +1074,7 @@ static const struct setting allowed_settings[] = {
         {
                 .name = "icon_position",
                 .section = "global",
-                .description = "Align icons left/right/off",
+                .description = "Default icon alignment (left/right/top/off)",
```

Comment:
In your checklist for changing a setting to a rule one of the bullets is
> Remove the setting from the global settings struct in settings.h`.

I have opted to keep `icon_position` as a global setting so that users can explicitly set a default value for all notifications. Consequently, if `icon_position` is unset in both cases, it will use the global setting's default.

Implemented here in the notification struct initialization:

https://github.com/dunst-project/dunst/blob/cfada4261f8883e849411c1a87959ac1d416df5b/src/notification.c#L421

Reply:
This should not be needed, since all rules also work in the global section (except for the filtering rules, since that would make the global section not global anymore).

Reply:
I see, I wasn't aware it worked that way. In that case I'll remove the global setting and its documentation